src/internal/components/dataset/NamedSetList.tsx

```diff
@@ -20,6 +20,8 @@ interface INamedSetListProps {
   value: INamedSet[] | null;
   onOpen: (event: React.MouseEvent<HTMLElement>, namedSet: INamedSet) => void;
   status: 'idle' | 'pending' | 'success' | 'error';
+  onDeleteNamedSet?: (namedSet: IStoredNamedSet) => void;
+  onEditNamedSet?: (namedSet: IStoredNamedSet) => void;
```

Comment:
Please add a comment that these callbacks are called at the end of the event listener. Otherwise one might wonder if it is executed before or after the actual change happens.

Reply:
Done.